### ISSUE TITLE
[NotABug] Build a REX memory image of the game, for easier loading on modern M100/T102 systems

### ISSUE THREAD

**Issue by majick** [ajbowen249/mol#10]
While the .CO file makes it somewhat easy to load the game from a TPDD emulator (or even a real TPDD!) to real hardware, even easier would be a REX memory image so that it's possible to simply switch to the game image and CALL it. Since tons of modern users of the M100/T102 have one or another kind of REX, this significantly increases the convenience.

Building a BY file is incredibly difficult to do programmatically during the build process so...

I just did it by hand and posted it to [my bucket](http://public.nachomountain.com/files/m100/). Feel free to grab the .BY file and post it as an artifact manually.

**Comment by ajbowen249** [comment]
Good idea! I went ahead and added your `.by` to [release 1.2.0](https://github.com/ajbowen249/mol/releases/tag/v1.2.0), for now.

**Comment by majick** [comment]
I've been trying to think of ways to actually do this during build; a couple of things came to mind:
1. Automate VitrualT to do it; there's some kind of automation API that might be capable of this, but I can't imagine it would be straightforward.
2. Start with a reference "blank" image from a running system — I have one, that's what I did on physical hardware — and overwrite the raw object file into place at wherever ORG is. This is something that could probably be done with raw shell/perl/python scripting.

I'd probably suggest the latter, even though it isn't a "clean" build. After all, blank the boot state of the M100 isn't exactly a moving target. 😆

**Comment by ajbowen249** [comment]
Option `2` has actually been floating around in my head 😆 I've got a branch of the engine going to explore ZX Spectrum support, and that community also has a "full image of the system" format that's pretty popular, so I could even have two platforms with that sort of build step.
